server/lib/pbench/process_tb.py

```diff
@@ -95,9 +102,13 @@ def process_tb(self) -> Results:
             tb = Path(str(tbmd5)[0 : -len(".md5")])
             tbdir = tb.parent
             controller = tbdir.name
+            satellite_prefix = ""
+
+            if "::" in controller:
+                satellite_prefix, controller = controller.split("::")
 
             try:
-                ProcessTb._results_push(controller, tb, self.token)
+                ProcessTb._results_push(tb, self.token, satellite_prefix)
```

Comment:
Looks like `controller` was dropped from this invocation of `_results_push()`, is that intentional?  The new call signature still seems to require it.